api/proto/teleport/legacy/types/types.proto

```diff
@@ -4985,6 +4987,19 @@ message PluginSlackAccessSettings {
   string fallback_channel = 1;
 }
 
+message PluginOpsgenieAccessSettings {
+  option (gogoproto.equal) = true;
+
+  // Address of Opsgenie
```

Comment:
Nit: These comments will get converted to godocs on the generated structs so they should follow Go convention.
```suggestion
  // Addr is the address of Opsgenie API.
```
Same for godocs below.